servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/RoundRobinToDefaultLBMigrationProvider.java

```diff
@@ -36,37 +35,24 @@
 public final class RoundRobinToDefaultLBMigrationProvider implements RoundRobinLoadBalancerBuilderProvider {
 
     static final String PROPERTY_NAME = "io.servicetalk.loadbalancer.roundRobinUsesDefaultLoadBalancer";
-    private static final boolean DEFAULT_PROPERTY_VALUE = false;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RoundRobinToDefaultLBMigrationProvider.class);
 
-    private final Properties properties;
-
-    public RoundRobinToDefaultLBMigrationProvider() {
-        this(System.getProperties());
-    }
-
-    // Exposed for testing.
-    RoundRobinToDefaultLBMigrationProvider(final Properties properties) {
-        this.properties = properties;
-    }
-
     @Override
     public <ResolvedAddress, C extends LoadBalancedConnection>
     RoundRobinLoadBalancerBuilder<ResolvedAddress, C> newBuilder(
             String id, RoundRobinLoadBalancerBuilder<ResolvedAddress, C> builder) {
         if (isEnabled()) {
-            LOGGER.info("Enabling DefaultLoadBalancer in place of RoundRobinLoadBalancer for client {}", id);
+            LOGGER.info("Enabling DefaultLoadBalancer in place of RoundRobinLoadBalancer");
```

Comment:
I would still include the `id` in the log message to see the difference between the default one and any user customization